### ISSUE TITLE
Editorial: Remove blank Benefits section from Understanding Guideline 2.5

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#2792]
In WCAG 2.2: The guidelines typically do not have a benefits section. [Guideline 2.5: Input Modalities Understanding](https://www.w3.org/WAI/WCAG22/Understanding/input-modalities) has a blank Benefits section which should be removed.

**Comment by kfranqueiro** [comment]
Fixed by #2822
